src/components/VolumeProperties.vue

```diff
@@ -87,6 +90,19 @@ export default defineComponent({
       <div ref="pwfEditorRef" />
     </div>
     <div v-if="!!cvrParams">
+      <v-slider
+        label="Volume Profile"
+        ticks="always"
+        tick-size="4"
+        :tick-labels="volumeQualityLabels"
+        hint="Hint"
```

Comment:
Add an appropriate hint or drop this attr.

Reply:
Ha!! Sorry about that. I was just playing with all the vue slider settings and left that in accidentally.